Source/Scene/Model.js

```diff
@@ -1543,6 +1682,10 @@ define([
             }
         }
 
+        if (usesExtension(model, 'WEB3D_quantized_attributes')) {
+            vs = modifyShaderForQuantizedAttributes(vs, id, model, context);
```

Comment:
> We could also add the uniforms to the glTF JSON in memory would I think would make createUniformsForQuantizedAttributes much simpler since it would be happen as the process over the glTF, not later when the commands are created.

[Online record] Discussed this and it doesn't make sense to do this because the quantized uniforms are per-primitive and trying to assign uniforms per mesh material as is required in gltf would require conflict resolution in meshes